app/code/BodenkoVV/AskQuestion/Controller/Adminhtml/Question/MassDisable.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace BodenkoVV\AskQuestion\Controller\Adminhtml\Question;
+
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Backend\App\Action\Context as ContextAlias;
+use Magento\Ui\Component\MassAction\Filter;
+use BodenkoVV\AskQuestion\Model\QuestionFactory;
+use BodenkoVV\AskQuestion\Model\ResourceModel\Question\CollectionFactory;
+
+/**
+ * Class MassDisable
+ */
+class MassDisable extends \Magento\Backend\App\Action
+{
+    /**
+     * Authorization level of a basic admin session
+     *
+     * @see _isAllowed()
+     */
+    const ADMIN_RESOURCE = 'BodenkoVV_AskQuestion::save';
+
+    /**
+     * @var Filter
+     */
+    protected $filter;
+
+    /**
+     * @var CollectionFactory
+     */
+    protected $_collectionFactory;
+
+
+    public $_questionFactory;
+
+    /**
+     * @param \Magento\Framework\Data\Form\FormKey\Validator $formKeyValidator
+     * @param ContextAlias $context
+     * @param Filter $filter
+     * @param CollectionFactory $collectionFactory
+     * @param QuestionFactory $questionFactory
+     */
+    public function __construct(
+        \Magento\Framework\Data\Form\FormKey\Validator $formKeyValidator,
+        ContextAlias $context,
+        Filter $filter,
+        CollectionFactory $collectionFactory,
+        QuestionFactory $questionFactory
+    )
+    {
+        //
+        $this->filter = $filter;
+        $this->_collectionFactory = $collectionFactory;
+        $this->_questionFactory = $questionFactory;
+        parent::__construct($context);
+    }
+
+    /**
+     * @return \DateTime $datetime
+     * @throws \Exception
+     */
+    public function getDatetimeNow() {
```

Comment:
Don't use new, please try injecting 
\Magento\Framework\Stdlib\DateTime\DateTime $date,
for example and get the date with the help of this
Please refer the details. https://magento.stackexchange.com/questions/89623/magento-2-get-current-store-date-time
Also the codestyle of this fragment has issues:
not 
```
 public function getDatetimeNow() {
```
but
``` 
public function getDatetimeNow() 
{
}
```
not $tz_object, but $timeZoneObject etc

Reply:
+